packages/block-editor/src/components/block-list/nav-up-icon.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * WordPress dependencies
+ */
+import { Icon, SVG, Path } from '@wordpress/components';
+
+const NavigateUp = <SVG width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><Path fill="none" d="M0 0h24v24H0V0z" /><Path fillRule="evenodd" clipRule="evenodd" d="M17,11  z L15.58,12.42 L12,8.83 L12,18 L22,18 L22,20 L10,20 L10,8.83 L6.42,12.42 L5,11 L11,5 L17,11" /></SVG>;
+
+export const NavigateUpSVG = <Icon icon={ NavigateUp } />;
```

Comment:
The pattern I've seen most often for icons is to make the `SVG` the default export and then do something like

```js
import icon from './icon'
```

Considering this will be passed to `ToolbarButton` -> `IconButton` -> `Icon`, I don't think we need to wrap it up in an `Icon` component here.

Reply:
done !